packages/server/lib/controllers/v1/flows/id/patchFrequency.ts

```diff
@@ -55,7 +55,7 @@ export const patchFlowFrequency = asyncWrapper<PatchFlowFrequency>(async (req, r
         return;
     }
 
-    const syncs = await getSyncsBySyncConfigId(environment.id, valParams.data.id);
+    const syncs = await getSyncsBySyncConfigId(valParams.data.id);
```

Comment:
Since we don't do permissions check yet and syncConfigId is auto-incremented it's not safe to query without env id. Happy to discuss but I would keep it especially since this value is available without join

Reply:
ok. I'll add it back

Reply:
[ae30d3e](https://github.com/NangoHQ/nango/pull/2709/commits/ae30d3ebea0799506c6abaea3dec2fb637001cc4)